binaryornot/check.py

```diff
@@ -31,3 +32,21 @@ def is_binary(filename):
     # Check if the starting chunk is a binary string
     chunk = get_starting_chunk(filename)
     return is_binary_string(chunk)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Check if a "
+                                                 "file passed as argument is "
+                                                 "binary or not")
+
+    parser.add_argument("filename", help="File name to check for. If "
+                                         "the file is not in the same folder, "
+                                         "include full path")
+
+    args = parser.parse_args()
+
+    print(is_binary(**vars(args)))
```

Comment:
This will print True or False literally, right? Is this the best way?

Reply:
What's a better response?